_posts/2017-10-23-atixag-shows-off-support-for-apt.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: AtixAG shows off support for Apt repos in Katello
```

Comment:
I'd like to see a space as in "Atix AG" here. And further down...

Reply:
Fixed, thanks.